Model/Media.php

```diff
@@ -618,7 +617,7 @@ public function getCategory()
     /**
      * @param mixed $category
      */
-    public function setCategory(CategoryInterface $category = null)
+    public function setCategory($category = null)
```

Comment:
the idea is to decouple the bundle. So that's why is required to remove the "Sonata\ClassificationBundle\Model\CategoryInterface"